backend/backend/main.py

```diff
@@ -3,13 +3,14 @@
 
 from dotenv import load_dotenv
 
-from api.routes import api_router
+from api.routes import api_router, gradio_routes
 from core.context import Context
 from utils.logger import logger
 
 from uuid import uuid4
-import os
 
+import gradio as gr
```

Comment:
Consider importing `gradio` and `os` at the top of the file for better organization.